database/index.js

```diff
@@ -12,33 +12,29 @@ db.connect((err) => {
   } else {
     console.log('Connected!');
   }
-})
-
-// !!!!!!!!!!!!!!!!! need id from url. From express i need to get req.query !!!!!!!!!!!!!!!!!!!
+});
 
-var insertToDB = (data, cb) => {
-  var query = `INSERT INTO photos (propertyID, url, description) VALUES (?, ?, ?)`; 
-  db.query(query, [data.propertyID, data.url, data.description], function(error, results, fields) {
+const insertToDB = (data, cb) => {
+  const query = 'INSERT INTO photos (propertyID, url, description) VALUES (?, ?, ?)';
+  db.query(query, [data.propertyID, data.url, data.description], (error, results) => {
     if (error) {
       cb(error);
       console.log(results);
     }
   });
-}
-
-// !!!!!!!!!!!!!!!!!! REDO QUERY STATEMENT TO CORRECT PHRASE !!!!!!!!!!!!!!!!!!
+};
 
-// var getFromDB = () => {
-//   var query = `SELECT * FROM photos WHERE propertyID ON ?`  ;
-//   db.query(query, [id], function(error, results, fields) {
-//     if (error) {
-//       callback(error, null);
-//     } else {
-//       callback(null, results);
-//     }
-//   });
-// }
+const getFromDB = (propertyID, callback) => {
+  const query = 'SELECT * FROM photos WHERE propertyID = ?';
+  db.query(query, propertyID, (error, results) => {
+    if (error) {
+      callback(error, null);
+    } else {
+      callback(null, results);
+    }
+  });
+};
 
 module.exports = db;
```

Comment:
This line is redundant, because it gets replaced by the export right below. You'll need to make an object to export both things.

Reply:
Acknowledged. I have deleted db export as I haven't imported elsewhere in my files.